src/lifecycles/mount.js

```diff
@@ -1,6 +1,6 @@
-import { NOT_MOUNTED, MOUNTED, SKIP_BECAUSE_BROKEN } from '../app.helpers.js';
-import { handleAppError, transformErr } from '../app-errors.js';
-import { reasonableTime } from '../timeouts.js';
+import { NOT_MOUNTED, MOUNTED, SKIP_BECAUSE_BROKEN } from '../applications/app.helpers.js';
+import { handleAppError, transformErr } from '../applications/app-errors.js';
+import { reasonableTime } from '../applications/timeouts.js';
```

Comment:
Just curious your thoughts here.
Parcels use the lifecycles the lifecycles import reasonableTime and additional functions from the applications subdirectory. Should we move these methods out of the subdirectory tied to applications only because they're used by both parcels and applications?

Reply:
Yes I think it might be nice to do that. I don't want this pr to turn into a massive refactor that moves everything, though, so let's do it in a follow-up pr.